### DISCUSSION TITLE
Access control

### DISCUSSION THREAD

**Discussion by yogeshm-jadhav** [payloadcms/payload#2674] [Q&A]
When I tried to access my website with different browser then it is not working.
or
How I can access collections or data without passing any credentials in my ReactJS website.

**Comment by DanRibbens** [comment]
The default access control for collections is that a user needs to be authenticated to create, read, update or delete. You can provide your own access function and simply return true in any collections that you want to be publicly readable. 

```ts
import { CollectionConfig } from 'payload/types';

const Posts: CollectionConfig = {
  slug: 'posts',
  access: {
    read: () => true,
  },
  fields: [ /*...*/]
}
```
